src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working");
+    }
+
+    public void stopWork() {
+        System.out.println("Excavator stopped working");
+

```

Comment:
use override if possible
```suggestion

    @Override
    public void stopWork() {
        System.out.println("Excavator stopped working");
```